app/controllers/jobseekers/confirmations_controller.rb

```diff
@@ -1,4 +1,26 @@
 class Jobseekers::ConfirmationsController < Devise::ConfirmationsController
+  # Overriden Devise method.
```

Comment:
I think this would be helpful if it becomes more generalised across the service and the documentation describes some general approach.

At the current state, I find it more beneficial to keep the explicit explanation of why this method is overridden and the branching comments  together with the code, so a developer in the future who is tempted to change this can have the context on why things are built in this way.

What I am going to do is to remove the last comment, that I find a little bit redundant now that I revisited this :scissors: